stable/coredns/templates/servicemonitor-autoscaler.yaml

```diff
@@ -1,21 +0,0 @@
-{{- if .Values.autoscaler.enabled }}
```

Comment:
So I was thinking to replace this line with and `{{- if and .Values.autoscaler.enabled .Values.prometheus.monitor.enabled
 }}` (cf. https://github.com/helm/charts/blob/f47e033003e45babf5375cf5437711298cb7ea2e/stable/coredns/templates/servicemonitor.yaml#L1 )?

Reply:
Turns out that the autoscaler doesn't expose any metrics 😢 So we don't need it for now.